djangocms_gmaps/cms_plugins.py

```diff
@@ -21,8 +21,9 @@ class LocationPlugin(CMSPluginBase):
     form = LocationForm
     module = settings.DJANGOCMS_GMAPS_PLUGIN_MODULE
     name = settings.DJANGOCMS_GMAPS_LOCATION_PLUGIN_NAME
-    render_plugin = False
+    render_plugin = True
```

Comment:
`render_plugin` by default is set to `True`

See http://docs.django-cms.org/en/latest/reference/plugins.html#cms.plugin_base.CMSPluginBase.render_plugin